Code/Framework/AzToolsFramework/AzToolsFramework/Slice/SliceDependencyBrowserComponent.cpp

```diff
@@ -208,9 +208,17 @@ namespace AzToolsFramework
     bool SliceDependencyBrowserComponent::GetSliceDependenciesByRelativeAssetPath(const AZStd::string& relativePath, AZStd::vector<AZStd::string>& dependencies) const
     {
         bool found = false;
+
+        AZ::Uuid sourceUuid;
+        m_databaseConnection->QuerySourceBySourceName(relativePath.c_str(), [&sourceUuid](AzToolsFramework::AssetDatabase::SourceDatabaseEntry& entry)
```

Comment:
should the logic look to see if sourceUuid was found before feeding it to the QueryDependsOnSourceBySourceDependency() call?

Reply:
I suppose it could, but in the off chance it came back with nothing, the next query should find nothing (since there really should not be an asset in the database with a null ID), and then it'd just return false the same way.